src/pages/dashboard/components/EpisodeDetails.tsx

```diff
@@ -66,10 +66,32 @@ function EpisodeDetails({ episode, isInCollection = false, showDate = false }: P
     return `${((resumePosition.asMilliseconds() / duration.asMilliseconds()) * 100).toFixed(2)}%`;
   }, [episode.Duration, episode.ResumePosition]);
 
+  const anidbEpisodePrefixes = (type: EpisodeTypeEnum, epNumber: number): string => {
```

Comment:
The last change I will request is to move the function out of `EpisodeDetails`, since it doesn't depend on the parent scope at all.

Reply:
I'm not quite understanding.
Is that you want me to move it out of `EpisodeDetails`'s component, or move it completely out of `EpisodeDetail`'s files, because both of them is count as out of `EpisodeDetails`'s scope?

Reply:
Just out of the component is fine. If you want to know where we place them in the file, then I will recommend looking at other files for optimal placement. The general rule of thumb is to place them above the main component (and any other sub-components inside the file).

From my understanding of the existing component files, then the order of declarations would be;
1. imports,
2. local types/interfaces,
3. local constants,
4. local helper functions, 
5. local helper components,
6. main component, and
7. export main component.